recipes/parm/meta.yaml

```diff
@@ -37,9 +37,15 @@ requirements:
 
 test:
   commands:
-    - parm --help
+    - parm train --help
 
 about:
   home: https://github.com/vansteensellab/PARM
   license: https://github.com/vansteensellab/PARM/blob/main/LICENSE
   summary: promoter activity regulatory model
+
+channels:
```

Comment:
Is this really needed ?

Reply:
I added this because, on the [anaconda page](https://anaconda.org/bioconda/parm), it doesn't show all the channels in the installation command.

I tried installing it with `conda install -c bioconda parm`, as it shows there, and I got a channel-related error. I can only install it with all the channels listed: `conda install -c anaconda -c conda-forge -c bioconda -c pytorch parm`. 

I thought adding the channels here in the yaml file, they would show up on the anaconda page.

Reply:
@martin-g should I remove those lines?

Reply:
I think you should remove them - https://bioconda.github.io/contributor/guidelines.html#dependencies

Reply:
Which dependency/ies is/are needed/used from `anaconda` and/or `pytorch` channels ?

Reply:
@martin-g h5py is used from `anaconda` channel, and pytorch is used from the `pytorch` channel

Reply:
okay, I'll remove them
thanks

Reply:
https://anaconda.org/search?q=h5py shows that version 3.11 is available in conda-forge channel
https://anaconda.org/search?q=pytorch - shows that version 2.3.1 is available in conda-forge. `pytorch` channel provides 2.4.0. Do you depend on something that is not available in 2.3.1 ?

Reply:
But https://anaconda.org/conda-forge/pytorch has a `broken` label ...
Let's see what the build will use!

Reply:
Oh, these versions should work fine.
Thanks again for your help!